src/commands/appdistribution-testers-remove.ts

```diff
@@ -0,0 +1,33 @@
+import {Command} from "../command";
+import * as utils from "../utils";
+import {requireAuth} from "../requireAuth";
+import {FirebaseError} from "../error";
+import {needProjectNumber} from '../projectUtils';
+import {AppDistributionClient} from '../appdistribution/client';
+import {getEmails} from '../testerEmailParser';
+
+
+module.exports = new Command("appdistribution:testers:remove [emails...]")
+    .description("Delete Testers")
+    .option(
+        "--file <file>",
+        "a path to a file containing a comma separated list of tester emails to be removed"
+    )
+    .before(requireAuth)
+    .action(async (emails: string[], options?: any) => {
+      const projectNumber = await needProjectNumber(options);
+      const request = new AppDistributionClient();
```

Comment:
Consider changing the variable name `request` to something more relevant.

Reply:
+1, and also do the same thing in the other command